build/nuget-release.yml

```diff
@@ -118,8 +118,8 @@ stages:
             parameters:
               repositoryName: 'arcus-azure/arcus.security'
               releaseNotes: |
-                Install new version via [NuGet](https://www.nuget.org/packages/Arcus.Security.All/$(Build.BuildNumber))
+                Install new version via [NuGet](https://www.nuget.org/packages/Arcus.Security.Providers.AzureKeyVault/$(Build.BuildNumber))
```

Comment:
I do wonder if we still need this task ?
As we are also calling the 'publish-official-packages' task, which pubishes nuget packages 

![image](https://user-images.githubusercontent.com/3605786/159082529-655c2afd-a4c6-44ef-bb58-0f2eb88ed14a.png)

I think it doesn't make sense to say 'install a new version via nuget' with a link to a 'random' default package.  Arcus.Security consists of a set of packages, and people need to decide themselves which packages are required.

Or, we should list them all here.

```
Install a new version of Arcus.Security.Core via [NuGet](link to Arcus.Security.Core)
Install a new version of Arcus.Security.AzureKeyVault via NuGet
etc...
```

Or, just a generic description that says:
> Install a new version of the Arcus.Security package that you want via NuGet

Reply:
(I think this is true for other Arcus packages as well).

Reply:
This is the GitHub release task, not the NuGet task. 
We could indeed be more detailed here instead of only describing one package. Though, the release notes will maybe a bit bloated if we do that.

Reply:
It is true that this is something we do across all Arcus libraries. So, if we plan on changing, we should track this on the general repo to change all libraries.

Reply:
I just think it makes no sense to mention a 'random' package from this project .

Reply:
It made more sense, maybe, when we still had that 'all' package.
I'll create an issue for this on the general repo, so we can discus it there.

Reply:
Added here: https://github.com/arcus-azure/arcus/issues/198

Reply:
I'll resolve this so that we can complete this, as it also makes no sense to currently refer to a non existing or unlisted package